src/test/java/org/openmrs/module/mirebalais/smoke/PatientRegistrationLiberiaFlowTest.java

```diff
@@ -67,6 +120,11 @@ protected Boolean getBiometricsEnabled() {
         return true;
     }
 
+    @Override
+    protected String getContactPhoneNumber() {
+        return "+231881112233";
+    }
+
```

Comment:
The idea is that you should *only* need to override this one method to change the phone number... if you need to do more than that we're doing something wrong...